.travis.yml

```diff
@@ -12,9 +12,11 @@ env:
     - DJANGO='1.8' REVERSION='1.9'  EXTRA=''
     - DJANGO='1.8' REVERSION='1.10' EXTRA=''
     - DJANGO='1.9' REVERSION='1.10' EXTRA=''
+    - DJANGO='1.9' REVERSION='2.0.4' EXTRA=''
     - DJANGO='1.8' REVERSION='1.9'  EXTRA='diff-match-patch'
-    - DJANGO='1.8' REVERSION='1.10'  EXTRA='diff-match-patch'
+    - DJANGO='1.8' REVERSION='1.10' EXTRA='diff-match-patch'
     - DJANGO='1.9' REVERSION='1.10' EXTRA='diff-match-patch'
+    - DJANGO='1.9' REVERSION='2.0.4' EXTRA='diff-match-patch'
```

Comment:
I think "2.0.4" is too limited. (Current django-reversion version is 2.0.6)

Why not use "2.0" ?